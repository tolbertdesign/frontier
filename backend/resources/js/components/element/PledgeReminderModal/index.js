import PledgeReminderModal from './PledgeReminderModal.vue'

export default PledgeReminderModal
