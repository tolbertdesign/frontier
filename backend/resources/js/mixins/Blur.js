export default {
  methods: {
    blur () {
      document.getElementById('app').style.filter = 'blur(4px)'
    },
    unBlur () {
      document.getElementById('app').style.filter = 'none'
    },
  },
}
