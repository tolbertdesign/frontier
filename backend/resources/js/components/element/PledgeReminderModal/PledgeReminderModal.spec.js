import { shallowMount, createLocalVue } from '@vue/test-utils'
import PledgeReminderModal from './PledgeReminderModal.vue'
import Blur from '@/mixins/Blur'

const localVue = createLocalVue()

const $store = {
  state: {
    lang: {
      send_explanation: 'send explanation',
      continue: 'continue',
      send: 'send',
      cancel: 'cancel',
      success: 'success',
      reminder_error: 'reminder error',
      close: 'close',
    },
  },
}

const pledgeId = 1

describe('PledgeReminderModal', () => {
  it('Snapshop test for PledgeReminderModal', () => {
    const wrapper = shallowMount(PledgeReminderModal, {
      localVue,
      mixins: {
        Blur,
      },
      mocks: {
        $store,
      },
      propsData: {
        pledgeId: pledgeId,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
