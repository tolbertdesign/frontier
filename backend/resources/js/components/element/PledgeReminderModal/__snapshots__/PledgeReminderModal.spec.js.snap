// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PledgeReminderModal Snapshop test for PledgeReminderModal 1`] = `
<form action="" class="mx-auto max-w-xs rounded-xl overflow-hidden shadow-lg bg-white">
  <div class="modal-card bg-white" style="width: auto;"><button type="button" aria-label="close" class="modal-close inner-modal-close modal-close-btn-position is-large absolute bg-transparent mb-4">Ã—</button>
    <p class="text-center text-lg leading-normal mt-12 max-w-xs">send explanation</p>
    <p class="text-center font-semibold text-lg leading-normal mt-4">continue</p>
    <div class="controls px-6">
      <div class="text-center mt-5 pb-1">
        <div class="max-w-md mx-0 md:mx-auto">
          <div><button type="submit" class="button is-secondary is-rounded inline-block mb-4 w-64 shadow">
              send
              <!----></button><br> <button type="button" class="button border-grey-dark text-grey-dark is-rounded inline-block mb-4 w-64 is-outlined shadow">
              cancel
            </button></div>
        </div>
      </div>
    </div>
  </div>
  <!---->
  <!---->
</form>
`;
